schemas/bill_summaries.py

```diff
@@ -0,0 +1,26 @@
+from pydantic import BaseModel, Field
+
+
+class BillSummaryUsSectionSchema(BaseModel):
+    """'bill_summary_us' section schema."""
+
+    text: str | None = Field(None)
+    id: str | None = Field(None)
```

Comment:
ditto

Reply:
`Field(None)` required because the field might be missing